src/client.ts

```diff
@@ -54,14 +56,23 @@ export interface JobOptions {
 export class Client {
   /** Internal transport used to send requests to Zeus. */
   private readonly transport: Transport;
+  /** GitHub repository owner */
+  public readonly owner: string;
+  /** GitHub repository name */
+  public readonly repo: string;
+  /** Path to directory where downloaded files are stored */
+  public readonly downloadDirectory: string;
```

Comment:
This should be called something like `tempDirectory` or `defaultDirectory`. Usually, the application will want to use explicit download locations every time it uses `downloadArtifact`, so it's better to clearly distinguish.